### DISCUSSION TITLE
MacOS Beta Abandoned?

### DISCUSSION THREAD

**Discussion by djmoore711** [notepadqq/notepadqq#1059] [General]
@danieleds I saw where you mentioned you didn't have the time for development about a year ago in a previous discussion. I assume that likely has not changed. I see the last macOS beta release was sometime in 2019. 

Do you know if anyone is taking over maintenance, or could you recommend a great alternative under active maintenance for macOS? 

Thanks for all your previous work. This looks like a great application and seems popular in the GNU/Linux community.

**Comment by danieleds** [comment]
hey @djmoore711, thank you :)

Unfortunately, I'm not aware of anyone maintaining the macOS release, at least officially. As usual, if someone would like to work on it, I can provide guidance and I'd be glad to help testing it.

I don't think there are text editors with a similar target/use case out there. I could suggest the usual big names, but I'm sure you are already aware of them.

For anyone reading: I'm also open to ideas on how to attract technical contributors. This has become a big project which requires collaboration and cooperation to stay alive. The only other sustainable alternative would be to make it a for-profit project (I need to pay the bills), but this is not gonna happen. I'm definitely _not_ going to put ads in it, or to sell data, or to make it closed-source with subscription plans. Never going to happen... open source cooperation is the way
